inference-engine/src/vpu/common/include/vpu/utils/small_vector.hpp

```diff
@@ -305,7 +305,11 @@ class SmallVector {
     const_reverse_iterator crend() const noexcept { return _base.crend(); }
 
     bool empty() const noexcept { return _base.empty(); }
-    size_type size() const noexcept { return _base.size(); }
+#if ENABLE_MYRIAD
+    int size() const noexcept { return static_cast<int>(_base.size()); }
+#else
+    size_t size() const noexcept { return _base.size(); }
+#endif
```

Comment:
why it should be different for ENABLE_VPU and other cases?

Reply:
This is to break dependency on plugin in different repository. else branch can be removed later.